src/products/firefox_desktop/review.md

```diff
@@ -4,140 +4,98 @@
 ```
 
 Check the results for duplicated strings and errors. For example, if a new error shows up for a missing variable, it’s likely that a string changed without a new ID and introduced new variables.
 
-The output of `compare-locales` against `mozilla-unified` is going to contain a lot of noise, since it includes all strings that are obsolete for `mozilla-central`, but are still needed for other branches. If you’re interested in only seeing missing strings, i.e. strings that need to be added to the l10n repository, you can `grep` the results by piping the output to `egrep '^\s*\+'`.
-
-```
-$ compare-locales --unified ~/src/mozilla-unified/browser/locales/l10n.toml ~/l10n it | egrep '^\s*\+'
-```
-
-### Push reviewed strings to gecko-strings
+You could also run `compare-locales` against the `gecko-dev` (git) or `mozilla-unified` (hg) repository. However, the output is going to contain a lot of noise, since it includes all strings that are obsolete for `mozilla-central`, but are still needed for other branches. If you’re interested in only seeing missing strings, i.e. strings that need to be added to the l10n repository, you can `grep` the results by piping the output to `egrep '^\s*\+'`.
 
-If there are no issues in `gecko-strings-quarantine` and no pending [Fluent migrations](../firefox_desktop/fluent_migrations.md), the next step is to push changes to `gecko-strings` and expose content to tools.
-
-Before pushing, [this script](https://github.com/flodolo/scripts/blob/main/mozilla_l10n/gecko_strings/gecko-quarantine-diff.py) can be used to compare the tip of the two repositories (it requires a local clone of `gecko-strings-quarantine`).
-
-One time setup: after you cloned `gecko-strings-quarantine` on your system, you need to edit its `.hg/hgrc` file, and add `gecko-strings` as path. While you only need `https` for pulling the quarantine repository, you need `ssh` in order to push updates to `gecko-strings`.
-
-The content of `~/l10n/gecko-strings-quarantine/.hg/hgrc` should be similar to this:
-
-```
-[paths]
-default = https://hg.mozilla.org/l10n/gecko-strings-quarantine
-gecko-strings = ssh://hg.mozilla.org/l10n/gecko-strings
 ```
-
-To push the current `default` to `gecko-strings`, from the `gecko-strings-quarantine` folder simply run:
-
-```
-$ hg push -r default gecko-strings
+$ compare-locales ~/mozilla-unified/browser/locales/l10n.toml ~/l10n it | egrep '^\s*\+'
 ```
 
-Instead of `default`, you can also push a specific changeset, e.g.
+### Merge reviewed strings to main branch of firefox-l10n-source
 
-```
-$ hg push -r 4c05bc050007 gecko-strings
-```
+If there are no issues in the `update` branch of `firefox-l10n-source` and no pending [Fluent migrations](../firefox_desktop/fluent_migrations.md), the next step is to merge the `update` branch into the `main` repository by approving and merging the [pull request](https://github.com/mozilla-l10n/firefox-l10n-source/pulls) in GitHub. Once merged this will expose content to tools.
```

Comment:
```suggestion
If there are no issues in the `update` branch of `firefox-l10n-source` and no pending [Fluent migrations](../firefox_desktop/fluent_migrations.md), the next step is to merge the `update` branch into the `main` repository by approving and merging the [pull request](https://github.com/mozilla-l10n/firefox-l10n-source/pulls) in GitHub. Once merged this will expose content to Pontoon.
```